src/components/Footer.jsx

```diff
@@ -3,65 +3,83 @@ import { Box, Flex, Text, Image, Wrap, Stack } from '@chakra-ui/react';
 const guys = [
   {
     name: 'Richard',
-    photo: './src/assets/smile.svg',
+    photo: './src/assets/images/richard-expert-footer.png',
   },
   {
     name: 'Gregory',
-    photo: './src/assets/smile.svg',
+    photo: './src/assets/images/greg-by-greegouze.png',
   },
   {
     name: 'Alexis',
-    photo: './src/assets/smile.svg',
+    photo: './src/assets/images/alexis-le-chercheur.png',
   },
   {
-    name: 'Typhen',
-    photo: './src/assets/smile.svg',
+    name: 'Thyphen',
+    photo: './src/assets/images/thyphen-vigile.png',
   },
 ];
 
+const SeparationLine = () => {
+  return (
+    <Flex
+      position='relative'
+      height={{ base: '1px', md: '125px' }}
+      w={{ base: '200px', md: '1px' }}
+      border='grey'
+      opacity='0.5'
+      bg='grey'
+      my='auto'
+      alignSelf={{ base: 'center', md: '' }}
+      top={{ base: '', md: '-25px' }}
+    />
+  );
+};
+
 const Footer = () => {
   return (
-    <Box as='footer' bg='black' w='100%' h={{ base: '480px', md: '250px' }}>
+    <Box as='footer' bg='gray.900' w='100%' h={{ base: '455px', md: '180px' }} overflow='hidden'>
       <Stack direction={['column', 'column', 'row', 'row']}>
         {/*Box authors-api */}
         <Flex
           direction='column'
           w={{ base: '100%', md: '50%' }}
-          gap='20px'
+          gap='10px'
           height={{ base: '290px', md: '250px' }}
           color='white'
-          p={{ base: '10px', md: '30px' }}
+          p={{ base: '10px', md: '10px' }}
         >
           {/*Box Authors title */}
           <Flex
-            justify={['center', 'center', 'flex-start']}
+            justify={['center', 'center', 'center']}
```

Comment:
Si c'est la même valeur, ça sert à rien de la mettre 3 fois
```suggestion
            justify={'center'}
```